Cloudflare/access_requests/ingest/parser.yml

```
@@ -0,0 +1,38 @@
+name: Cloudflare Access Requests
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+  - name: set_ecs_fields
+  - name: set_cloudflare_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          "@timestamp": '{{json_event.message.CreatedAt}}'
+          client.ip: '{{json_event.message.IPAddress}}'
+          event.category: ["network", "authentication"]
+          event.dataset: "access_request"
+          event.kind: 'event'
+      - set:
+          event.type: ["connection", "start"]
+        filter: '{{json_event.message.Action == "login" and json_event.message.Allowed == "allowed"}}'
+      - set:
+          event.type: ["connection","end"]
+        filter: '{{json_event.message.Action == "logout"}}'
+      - set:
+          event.type: ["connection","denied"]
+        filter: '{{json_event.message.Allowed == "denied"}}'
+      - set:
+          url.full: '{{json_event.message.AppDomain}}'
```

I suggest using `url.domain` instead (and benefit from the url finalization):
```suggestion
          url.domain: '{{json_event.message.AppDomain}}'
```